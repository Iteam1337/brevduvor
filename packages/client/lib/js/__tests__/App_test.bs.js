// Generated by BUCKLESCRIPT VERSION 5.0.6, PLEASE EDIT WITH CARE
'use strict'

var Jest = require('@glennsl/bs-jest/lib/js/src/jest.js')
var React = require('react')
var App$Client = require('../src/App.bs.js')
var TestUtils$Client = require('../src/TestUtils.bs.js')
var ReactTestingLibrary = require('bs-react-testing-library/lib/js/src/ReactTestingLibrary.bs.js')

Jest.describe('App', function(param) {
  return Jest.test('renders', function(param) {
    var component = React.createElement(
      TestUtils$Client.MockedProvider[/* make */ 0],
      {
        children: React.createElement(App$Client.make, {}),
      }
    )
    return Jest.Expect[/* toMatchSnapshot */ 16](
      Jest.Expect[/* expect */ 0](
        ReactTestingLibrary.render(undefined, undefined, component).container
      )
    )
  })
})

/*  Not a pure module */
