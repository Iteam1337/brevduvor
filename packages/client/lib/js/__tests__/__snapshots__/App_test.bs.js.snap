// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders login screen initially 1`] = `
<div>
  <div>
    <div
      class="flex fixed bg-background w-full min-h-screen z-50 items-center justify-center"
    >
      <div
        class="w-full max-w-xs"
      >
        <form
          class="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4"
        >
          <div
            class="mb-4"
          >
            <div
              class="mb-4"
            >
              <label
                class="block text-gray-700 text-sm font-bold mb-2"
                for="email"
              >
                E-post
              </label>
              <input
                class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
                id="email"
                placeholder="min@epost.se"
                type="email"
              />
            </div>
          </div>
          <div
            class="mb-6"
          >
            <div
              class="mb-4"
            >
              <label
                class="block text-gray-700 text-sm font-bold mb-2"
                for="password"
              >
                LÃ¶senord
              </label>
              <input
                class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
                id="password"
                placeholder="********"
                type="Password"
              />
            </div>
          </div>
          <button
            class="w-full text-xs bg-blue-400 hover:bg-blue-500 text-white font-semibold
        py-3 px-4 rounded tracking-wide border border-blue-400 hover:border-blue-500 mt-5 w-full text-xs bg-blue-400 hover:bg-blue-500 text-white font-semibold
        py-3 px-4 rounded tracking-wide border border-blue-400 hover:border-blue-500"
            type="submit"
          >
            Logga in
          </button>
        </form>
      </div>
    </div>
    <div
      class="absolute z-10 top-4 right-4"
    />
  </div>
</div>
`;
